client/components/command-palette/index.tsx

```diff
@@ -80,7 +80,9 @@ export function CommandMenuGroup( {
 	const { commands } = useCommandPallette( {
 		selectedCommandName,
 		setSelectedCommandName,
-		filter: isContextual ? ( command ) => command.context?.includes( currentPath ) : undefined,
+		filter: isContextual
+			? ( command ) => command.context?.some( ( path ) => currentPath.includes( path ) ) ?? false
+			: undefined,
```

Comment:
I love this function:
`( command ) => command.context?.some( ( path ) => currentPath.includes( path ) )`
It's pure magic.

I think we can remove `isContextual` and also `filter`. And define the given function inside the `useCommandPallette` hook.

Reply:
Sounds good, I have removed it and moved the logic of `filter` into the `commandHasContext` function

Reply:
Yeah, I learned about `array.prototype.some` today: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some ❤️